pyro/infer/mcmc/mcmc.py

```diff
@@ -272,6 +272,9 @@ def __init__(self, kernel, num_samples, warmup_steps=None,
         else:
             self.sampler = _SingleSampler(kernel, num_samples, self.warmup_steps, disable_progbar)
         super(MCMC, self).__init__(num_chains=num_chains)
+        warnings.warn("This interface to MCMC is deprecated and will be removed in the "
+                      "next version of Pyro. Please use `pyro.infer.mcmc.api` instead.",
+                      DeprecationWarning)
```

Comment:
So if I understand correctly, then when we remove this interface, we can use `infer.mcmc.MCMC` with the new api?

Reply:
We can add this to `infer.mcmc.__init__.py` so that this will be the default interface.